python/nano/src/bigdl/nano/tf/keras/Model.py

```diff
@@ -13,11 +13,54 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
-
 import tensorflow as tf
 
 
 class Model(tf.keras.Model):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+
+    def fit(self,
+            x=None,
+            y=None,
+            batch_size=None,
+            epochs=1,
+            verbose='auto',
+            callbacks=None,
+            validation_split=0.,
+            validation_data=None,
+            shuffle=True,
+            class_weight=None,
+            sample_weight=None,
+            initial_epoch=0,
+            steps_per_epoch=None,
+            validation_steps=None,
+            validation_batch_size=None,
+            validation_freq=1,
+            max_queue_size=10,
+            workers=1,
+            use_multiprocessing=False,
+            perf_tune=None):
+        if perf_tune is not None:
```

Comment:
How about making perf_tune a boolean argument, and we will tuning all of `batch_size`, `callbacks` and `dataset`

Reply:
Or I add an option "All" to run all of the tunning task.